<?xml version="1.0"?>

<launch>
  <node pkg="detection" type="detection" name="detection_node" output="screen" >
    <param name="engine_file" value="$(find detection)/model/oap_mobilenet.engine" />
    <param name="imwrite_path" value="$(find detection)/imwrite/" /> 
    
    <param name="is_visulized" value="1" /> 

  </node>

</launch>
